src/functions.date.js

```diff
@@ -94,6 +94,20 @@ $.extend(_, {
 	}, {multiValued: true}),
 
 	localTimezone: -(new Date()).getTimezoneOffset(),
+	getduration: $.extend(function (dur, format) {
+		if(format === "officialDurationString") {durationRegEx = /P(?:([.,\d]+)D)?(?:T(?:([.,\d]+)H)?(?:([.,\d]+)M)?(?:([.,\d]+)S)?)?/;}
```

Comment:
Please install ESLint + an ESLint plugin for your editor so you can get inline warnings when the coding style of the project is not followed.